src/GraphQL.Tests/Validation/OverlappingFieldsCanBeMergedTest.cs

```diff
@@ -255,7 +255,7 @@ public void Allows_different_args_where_no_conflict_is_possible_should_pass()
         const string query = """
             fragment conflictingArgs on Pet {
                 ... on Dog {
-                    name(surname: true)
+                    name(surname: "test")
```

Comment:
This test needed to be changed because `surname` is defined as a string, and is now validated along with the document before the field resolver hits.  The boolean value failed validation before.